src/main/target/RACEBASE/config.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdint.h>
+
+#include <platform.h>
+
+#include "build_config.h"
+
+#include "blackbox/blackbox_io.h"
+
+#include "common/color.h"
+#include "common/axis.h"
+#include "common/filter.h"
+
+#include "drivers/sensor.h"
+#include "drivers/accgyro.h"
+#include "drivers/compass.h"
+#include "drivers/system.h"
+#include "drivers/timer.h"
+#include "drivers/pwm_rx.h"
+#include "drivers/serial.h"
+#include "drivers/pwm_output.h"
+#include "drivers/max7456.h"
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+
+#include "sensors/sensors.h"
+#include "sensors/gyro.h"
+#include "sensors/compass.h"
+#include "sensors/acceleration.h"
+#include "sensors/barometer.h"
+#include "sensors/boardalignment.h"
+#include "sensors/battery.h"
+
+#include "io/beeper.h"
+#include "io/serial.h"
+#include "io/gimbal.h"
+#include "io/escservo.h"
+#include "io/rc_controls.h"
+#include "io/rc_curves.h"
+#include "io/ledstrip.h"
+#include "io/gps.h"
+#include "io/osd.h"
+#include "io/vtx.h"
+
+#include "rx/rx.h"
+
+#include "telemetry/telemetry.h"
+
+#include "flight/mixer.h"
+#include "flight/pid.h"
+#include "flight/imu.h"
+#include "flight/failsafe.h"
+#include "flight/altitudehold.h"
+#include "flight/navigation.h"
+
+#include "config/runtime_config.h"
+#include "config/config.h"
+
+#include "config/config_profile.h"
+#include "config/config_master.h"
+
+// alternative defaults settings for COLIBRI RACE targets
+void targetConfiguration(master_t *config) {
+    config->rxConfig.sbus_inversion = 0;
+    config->rxConfig.rssi_scale = 19;
+    config->escAndServoConfig.minthrottle = 1060;
```

Comment:
Hmm, yes, maybe this lines:

`    config->escAndServoConfig.minthrottle = 1060;`
`    config->escAndServoConfig.maxthrottle = 1940;`

is not good idea because it depend on pilot, but other settings is for this board only so I think we can keep it. Remove this two lines, or you will remove it when merge it ?